definitions/private-law/json/AuthorisationCaseField/DraftAnOrder/AuthorisationCaseField.json

```diff
@@ -89,5 +89,30 @@
     "CaseFieldID": "draftOrderCollection",
     "UserRole": "caseworker-privatelaw-judge",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "10/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "c43Label",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-privatelaw-systemupdate",
+          "caseworker-wa-task-configuration",
+          "caseworker-privatelaw-solicitor"
```

Comment:
can we be specific here. we need access for applicant solicitor, also respondent solicitor. As we are moving away from IDAM to case roles, it needs to be specific

Reply:
Have changed caseworker-privatelaw-solicitor to all the specific roles.